data/saas/config/mandrill_config.yml

```diff
@@ -0,0 +1,44 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Mandril SaaS Config
+  type: mandrill
+  description: A sample schema representing the Mandril connector for Fided
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: api_key
+
+  client_config:
+    protocol: https
+    host: <domain>
+
+  consent_requests:
+    opt_in:
+      - method: POST
+        path: /allowlists/add
+        body: |
+          {
+            "key": "<api_key>",
+            "email": "<email>"
+          }
+        data_uses: [advertising.first_party]
+    opt_out:
+      - method: POST
+        path: /allowlists/delete
+        body: |
+          {
+            "key": "<api_key>",
+            "email": "<email>"
+          }
+        data_uses: [ advertising.first_party]
```

Comment:
To me it seems like the saas config needs to hold the distinction between:
1) Whether the endpoint is for an opt_in or opt_out request
2) Which data_uses apply

Reply:
leave opt in and opt out sections remove data uses